### DISCUSSION TITLE
Managing X2Go desktop sessions

### DISCUSSION THREAD

**Discussion by dashohoxha** [epoptes/epoptes#92] [Q&A]
Epoptes does not seem to recognize and manage desktop views of users that use X2Go to access the server. Maybe something needs to be fixed or improved.

**Comment by alkisg** [comment]
Can you manually run `epoptes-client` in a terminal, while inside an x2go session, and see if this makes it work?

If so, it would be an issue with x2go not running /etc/xdg/autostart items.

**Comment by alkisg** [comment]
No feedback, closing.

**Comment by dashohoxha** [comment]
@alkisg you are right. If autostart items run correctly everything works fine. And not only with X2Go sessions, but also with Guacamole sessions (I just tested it).
